client/src/drawerTabs/ShareSettings.tsx

```diff
@@ -107,12 +106,16 @@ export function ShareSettings({
   allLicenses: License[];
   remixedWithLicense: LicenseCode | null;
 }) {
-  const license = contentData.license;
+  const licenseCode = contentData.licenseCode;
 
   const [selectedIsPublic, setSelectedIsPublic] = useState(
     contentData.isPublic,
   );
-  const [selectedLicenseCode, setSelectedLicenseCode] = useState(license?.code);
+
+  // TO REVIEW: Is this logic ok?
+  const [selectedLicenseCode, setSelectedLicenseCode] = useState(
```

Comment:
Unfortunately, we are going to have legacy content from doenet.org where the author did not assign a license. That is why we leave the license code field optional, even though it is required for new content. We can't assign any license to that content if an author never licensed it, so we have to deal with the case of no license.

Maybe eventually, we'll just stop sharing content without a license and make the license code required.